.circleci/config.yml

```diff
@@ -18,3 +18,37 @@ jobs:
             - ./node_modules
       - run: npm run lint
       - run: npm test
+      - store_test_results:
+          path: ./reports/junit
+      - store_artifacts:
+          path: ./reports/junit
```

Comment:
This project isn't generating these files yet but I'll make sure they do in my other pending PR.